### DISCUSSION TITLE
Bluetooth dongle Raspberry 1

### DISCUSSION THREAD

**Discussion by nasp2000** [rern/rAudio#451] [General]
Hi, i have tried to add a bluetooth dongle to the first version of Raspberry but it not show on rAudio web interface like the bluetooth onboard like RPi3.

How i can add a bluetooth in this case ?

**Comment by R101G** [comment]
Built-in support for wireless dongles is limited, so you will have to check if yours is supported and install the driver if it isn't. Start with the _lsusb_ command, which will show the chipset I.D. If the command does not work, install it with _pacman -S usbutils_.
